script.js

```diff
@@ -31,3 +31,82 @@ document.addEventListener('scroll', () => {
+    imgURL: './assets/Snapshoot-Portfolio-4.png',
+    heading: 'Multi-Post Stories',
+    firstList: ['CANOPY', 'Back End', '2015'],
+    text: 'A daily selection of privately personalized reads; no accounts or sign-ups required. Lorem ipsum dolor sit amet consectetur adipisicing elit.',
+    secList: ['HTML', 'CSS', 'JavaScript'],
+  },
+  {
+    id: 1,
+    imgURL: './assets/Snapshoot-Portfolio.png',
+    heading: 'Facebook 360',
+    firstList: ['FACEBOOK', 'FULL Stack Dev', '2015'],
+    text: 'A daily selection of privately personalized reads; no accounts or sign-ups required. Lorem ipsum dolor sit amet consectetur adipisicing elit.',
+    secList: ['HTML', 'CSS', 'JavaScript', 'Ruby on rails'],
+  },
+  {
+    id: 2,
+    imgURL: './assets/Snapshoot-Portfolio-1.png',
+    heading: 'Uber Navigations',
+    firstList: ['UBER', 'Lead Developer', '2018'],
+    text: 'A daily selection of privately personalized reads; no accounts or sign-ups required. Lorem ipsum dolor sit amet consectetur adipisicing elit.',
+    secList: ['HTML', 'CSS', 'JavaScript', 'Ruby on rails'],
+  },
+  {
+    id: 3,
+    imgURL: './assets/Snapshoot-Portfolio-2.png',
+    heading: 'Multi-Post Stories',
+    firstList: ['CANOPY', 'Back End', '2015'],
+    text: 'A daily selection of privately personalized reads; no accounts or sign-ups required. Lorem ipsum dolor sit amet consectetur adipisicing elit.',
+    secList: ['HTML', 'CSS', 'JavaScript', 'Ruby on rails'],
+  },
+  {
+    id: 4,
+    imgURL: './assets/Snapshoot-Portfolio-3.png',
+    heading: 'Multi-Post Stories',
+    firstList: ['FACEBOOK', 'FULL Stack Dev', '2015'],
+    text: 'A daily selection of privately personalized reads; no accounts or sign-ups required. Lorem ipsum dolor sit amet consectetur adipisicing elit.',
+    secList: ['HTML', 'CSS', 'JavaScript', 'Ruby on rails'],
+  },
+];
+
+const closePopup = document.querySelector('#popup-head > span');
+const PopUp = document.querySelector('#pop-up');
+const cards = document.querySelectorAll('.card a');
+
+closePopup.addEventListener('click', () => {
+  PopUp.classList.toggle('hidden');
+});
+
+const makePopup = ({
+  imgURL, heading, firstList, text, secList,
+}) => {
+  const pHead = document.querySelector('#popup-head > h2');
+  const P1List = document.querySelectorAll('#pop-up ul > li');
+  const PImg = document.querySelector('#popup-img > img');
+  const PText = document.querySelector('#popup-details > p');
+  const P2List = document.querySelector('#popup-details menu');
+
+  pHead.innerHTML = heading;
+  PText.innerHTML = text;
+  PImg.src = imgURL;
+  P1List.forEach((ele, i) => { ele.innerHTML = firstList[i]; });
+  P2List.innerHTML = '';
+  secList.forEach((ele) => {
+    const listItem = document.createElement('li');
+    listItem.innerHTML = ele;
+    P2List.appendChild(listItem);
+  });
+  PopUp.classList.toggle('hidden');
+};
+
+cards.forEach((ele, i) => {
+  ele.addEventListener('click', () => {
+    makePopup(projectArray[i]);
+  });
+});
```

Comment:
- [ ] Great job 👍 . Immediately your website loads the pop-up displays by default. It is meant to display only when the user clicks on the button please fix this bug 🙏